cumulusci/tasks/bulkdata/load.py

```diff
@@ -113,6 +113,7 @@ def _load_mapping(self, mapping):
         if mapping.get("fields", {}).get("RecordTypeId"):
             conn = self.session.connection()
             self._load_record_types([mapping["sf_object"]], conn)
+            self.session.commit()
```

Comment:
Should this be in a try/finally block?

Reply:
Why would we want to commit if there was an exception?